client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -255,11 +259,14 @@ class CancelPurchase extends Component {
 
 export default connect( ( state, props ) => {
 	const purchase = getByPurchaseId( state, props.purchaseId );
+	const isJetpackPurchase =
+		purchase && ( isJetpackPlan( purchase ) || isJetpackProduct( purchase ) );
```

Comment:
Since this is quite used in various other places, I'd recommend having it as a helper function. However, that's completely out of the scope of this change.